code/game/mecha/working/ripley.dm

```diff
@@ -33,7 +33,7 @@
 		var/obj/structure/ore_box/ore_box = locate(/obj/structure/ore_box) in cargo
 		if(ore_box)
 			for(var/obj/item/stack/ore/ore in range(1, src))
-				if(ore.Adjacent(src) && ((get_dir(src, ore) & dir) || ore.loc == loc)) //we can reach it and it's in front of us? grab it!
+				if(ore.Adjacent(src) || ore.loc == loc) //we can reach it? grab it!  austation code  Removed the necessity for the ores to be in front of the mech
```

Comment:
```suggestion
				if(ore.Adjacent(src) || ore.loc == loc) //austation — Removed the necessity for the ores to be in front of the mech
```

Reply:
Phone is turning my hyphens into an em-dash but you get the point

Reply:
The beginning of that comment is from the original author and explains what the line is supposed to achieve. I don't see why we should remove it.